src/drivers/code_hosting/gitlab.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local branch=$2
+  local parent_branch=$3
+
+  local to_compare
+  if [ "$parent_branch" = "$MAIN_BRANCH_NAME" ]; then
+    # Allow GitLab to redirect to the proper place if this repository is a fork
+    to_compare="$branch"
```

Comment:
thanks @charlierudolph 

it's odd that some of the tests passed previously. Regardless, thanks to your help I was able to identify what was missing and have updated the implementation.

I still see sporadic test failures. But this is not consistent between runs, nor across many runs. I suspect my laptop get's a little unhappy running the full suite (the fans certainly make enough noise!)

If there's anything amiss let me know and I'll do my best to address it